server/src/test/java/io/crate/metadata/doc/mappers/array/ArrayMapperTest.java

```diff
@@ -442,47 +442,4 @@ public void testParseDynamicNullArray() throws Exception {
         assertThat(doc.doc().getField("new_array_field"), is(nullValue()));
         assertThat(mapper.mappers().getMapper("new_array_field"), is(nullValue()));
     }
-
-    @Test
-    public void testCopyToFieldsOfInnerMapping() throws Exception {
-        // @formatter:off
-        String mapping = Strings.toString(JsonXContent.builder()
-            .startObject().startObject(TYPE).startObject("properties")
-                .startObject("string_array")
-                    .field("type", ArrayMapper.CONTENT_TYPE)
-                    .startObject(ArrayMapper.INNER_TYPE)
-                        .field("type", "text")
-                        .field("index", "true")
-                        .field("position", 1)
-                        .field("copy_to", "string_array_ft")
-                    .endObject()
-                .endObject()
-            .endObject().endObject().endObject());
-        // @formatter:on
-
-        DocumentMapper mapper = mapper(INDEX, mapping);
-        FieldMapper arrayMapper = (FieldMapper) mapper.mappers().getMapper("string_array");
-        assertThat(arrayMapper, is(instanceOf(ArrayMapper.class)));
-        assertThat(arrayMapper.copyTo().copyToFields(), contains("string_array_ft"));
-
-        // @formatter:on
-        BytesReference bytesReference = BytesReference.bytes(JsonXContent.builder()
-            .startObject()
-            .startArray("string_array")
-            .value("foo")
-            .value("bar")
-            .endArray()
-            .endObject());
-        SourceToParse sourceToParse = new SourceToParse(INDEX, "1", bytesReference, XContentType.JSON);
```

Comment:
We already have an integration test `FulltextIntegrationTest.testCopyValuesFromStringArrayToIndex` for that case.
This test and the Itest ^ were added in https://github.com/crate/crate/commit/2f9e75f2c1ab9ddd3f393796424ed3793d1343c4.